e2e/testconfig/testconfig.go

```diff
@@ -8,42 +8,65 @@ import (
 )
 
 const (
-	DefaultSimdImage = "ghcr.io/cosmos/ibc-go-simd-e2e"
-	SimdImageEnv     = "SIMD_IMAGE"
-	SimdTagEnv       = "SIMD_TAG"
-	GoRelayerTag     = "RLY_TAG"
+	ChainASimdImageEnv = "CHAIN_A_SIMD_IMAGE"
+	ChainASimdTag      = "CHAIN_A_SIMD_TAG"
+	ChainBSimdImageEnv = "CHAIN_B_SIMD_IMAGE"
+	ChainBSimdTagEnv   = "CHAIN_B_SIMD_TAG"
+	GoRelayerTagEnv    = "RLY_TAG"
 
-	defaultRlyTag = "main"
+	defaultSimdImage = "ghcr.io/cosmos/ibc-go-simd-e2e"
+	defaultRlyTag    = "main"
 )
 
 // TestConfig holds various fields used in the E2E tests.
 type TestConfig struct {
-	SimdImage string
-	SimdTag   string
-	RlyTag    string
+	ChainAConfig ChainConfig
+	ChainBConfig ChainConfig
+	RlyTag       string
+}
+
+type ChainConfig struct {
+	Image string
+	Tag   string
 }
 
 // FromEnv returns a TestConfig constructed from environment variables.
 func FromEnv() TestConfig {
-	simdImage, ok := os.LookupEnv(SimdImageEnv)
+	chainASimdImage, ok := os.LookupEnv(ChainASimdImageEnv)
+	if !ok {
+		chainASimdImage = defaultSimdImage
+	}
+
+	chainASimdTag, ok := os.LookupEnv(ChainASimdTag)
 	if !ok {
-		simdImage = DefaultSimdImage
+		panic(fmt.Sprintf("must specify simd version for test with environment variable [%s]", ChainASimdTag))
 	}
 
-	simdTag, ok := os.LookupEnv(SimdTagEnv)
+	chainBSimdImage, ok := os.LookupEnv(ChainBSimdImageEnv)
 	if !ok {
-		panic(fmt.Sprintf("must specify simd version for test with environment variable [%s]", SimdTagEnv))
+		chainBSimdImage = chainASimdImage
 	}
 
-	rlyTag, ok := os.LookupEnv(GoRelayerTag)
+	chainBSimdTag, ok := os.LookupEnv(ChainBSimdTagEnv)
+	if !ok {
+		chainBSimdTag = chainASimdTag
```

Comment:
Should we document the default behaviour somewhere? That is when the chainB image/tag isn't specified. It might be a little odd if the image is specified but that tag isn't? or vice versa?